BOJ/1011/ghwns82.py

```diff
@@ -0,0 +1,33 @@
+import math
+for _ in range(int(input())):
+    start, end = map(int, input().split())
+    distance = end -start
+    max_spped = math.floor(math.sqrt(distance))
```

Comment:
max_speed 였던건가?
```suggestion
    max_speed = int(math.sqrt(distance))
```
그리고 int 연산이랑 수학 모듈 내림 연산이랑 뭔가 차이 있나?

Reply:
찾아보니 큰 차이 없네